e2e/scripts/wait-for-ready-nodes.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# This script waits for X nodes to be ready. X is the first argument.
+
```

Comment:
ditto re `set -euo pipefail`

Reply:
As this has been mentioned multiple times I am closing this to keep the discussion [here](https://github.com/replicatedhq/helmbin/pull/31#discussion_r1298473417).